rpc/client_test.go

```diff
@@ -34,6 +35,20 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+func TestParseRawURL(t *testing.T) {
```

Comment:
DELETEME

Reply:
I will do this.

Reply:
https://github.com/etclabscore/core-geth/pull/168/commits/2849633505f328276693dcd6ccb894625e90b114